### ISSUE TITLE
Update DEPRECATED dependencies ?

### ISSUE THREAD

**Issue by ayunawani** [Gongreg/storybook-usage#1]
Hi @Gongreg ,
Just wanted to know, will you update your all `@kadira` dependencies to [@storybook](https://github.com/storybooks/storybook) 
 dependencies ? 

for e.g : 
```
    "@kadira/storybook-addons": "^1.3.1",
    "@kadira/storybook-ui": "^3.2.0",
```
to

```
    "@storybook/addons":  "^3.0.0-alpha.0"
    "@storybook/ui" :  "^3.0.0-alpha.0"
```

**Comment by Gongreg** [comment]
@ayunawani, I can do it,  give me a day or two :)

**Comment by Gongreg** [comment]
Hey, sorry that I took so long, I was sick so I had no chance to work on it.

**Comment by Gongreg** [comment]
@ayunawani Hey, I released version 3.0.0 to npm. I did major just in case somebody want to use older dependencies (I know that nobody will use it but still :) ). Can you reply after you try it out if everything works correctly?

**Comment by ayunawani** [comment]
Hey @Gongreg, 
Thanks for the new release, its working great . 👍
